bin/ocamlformat-rpc/main.ml

```diff
@@ -143,7 +156,22 @@ let rec rpc_main = function
             V1.Command.output stdout (`Config c) ;
             Out_channel.flush stdout ;
             rpc_main (Version_defined (v, conf))
-        | Error e -> handle_config_error e ; rpc_main state ) ) )
+        | Error e -> handle_config_error e ; rpc_main state ) )
+    | V2 -> (
+      match V2.Command.read_input stdin with
+      | `Halt -> Ok ()
+      | `Unknown | `Error _ -> rpc_main state
+      | `Format (x, format_args) ->
+          let conf =
+            match run_format_with_args format_args conf x with
+            | Ok (`Format formatted) ->
+                ignore (Format.flush_str_formatter ()) ;
```

Comment:
Is it needed ? If it is, perhaps it should be done in `Translation_unit`.

Reply:
Apparently it's not, I'm removing this.